### ISSUE TITLE
Ensure Accessible Font Sizes

### ISSUE THREAD

**Issue by chznbaum** [757colorcoded/wp-colorcoded#3]
For accessibility reasons, it's best not to have text below 14px (most text should be 16px or higher). Using browser developer tools, such as [Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/) or [Firefox Developer Tools](https://developer.mozilla.org/en-US/docs/Tools), please find places where text is too small, and apply a CSS rule to set a larger font size.

Multiple pull requests are welcomed as each pull request can include 1+ size changes.

**Comment by yati1998** [comment]
I would like to solve this
